module/actor.js

```diff
@@ -606,6 +606,25 @@ export class cyberpunkredActor extends Actor {
     }
   }
 
+  isCritInjury(roll) {
```

Comment:
I just realized I defined these functions in an `actor` class, and `actor.isCritInjury` might be confusing. Think I should rename them to something like `checkCritInjury` instead, or move them into a different class?